lib/openqa.rb

```diff
@@ -55,7 +55,7 @@ def notify
 
     logger.info('openQA job done, notifying...')
 
-    notify_mail if job['result'].in?(%w[passed failed])
+    notify_mail if %w[passed failed].include?(job['result'])
```

Comment:
I wonder if this is going to fix the error we see in the log. 
For sure the change doesn't harm as both lines are equivalent, however, using `include?` avoids some issues. See the example:

```
[7] pry(main)> %w[passed failed].include?("failed":String)
=> false
[8] pry(main)> "failed":String.in?(%w[passed failed])
SyntaxError: unexpected ':', expecting end-of-input
"failed":String.in?(%w[passed failed])
        ^
```

It confuses me that the error I get in the Rails console is different from the log one.